terraform/modules/spack/karpenter.tf

```diff
@@ -70,6 +70,12 @@ resource "helm_release" "karpenter" {
     value = "1Gi"
   }
 
+  # Enable service monitor for prometheus metrics
+  set {
+    name = "serviceMonitor.enabled"
+    value = true
+  }
```

Comment:
Have you tested this at all to confirm there's no other config required?

Reply:
Yes, I did a terraform apply in staging and it works as expected.